clar2wasm/src/words/control_flow.rs

```diff
@@ -295,6 +295,16 @@ mod tests {
 
     use crate::tools::{crosscheck, crosscheck_expect_failure, evaluate};
 
+    #[test]
+    fn begin_consecutive_statements() {
```

Comment:
If we end up keeping this test, there should be a comment indicating that this is not due to clarity-wasm. 
Imagine being a newcomer to this repo and seeing this test fail. My reaction would be to start debugging everything possible in the word `begin`, instead of thinking that there is a bug in our dependencies.

Reply:
I am starting to being sold to your argument. :-D

Let's keep only the commit to silence the warning on this PR.